src/bootstrap/config.toml.example

```diff
@@ -152,6 +152,9 @@
 # known-good version of OpenSSL, compile it, and link it to Cargo.
 #openssl-static = false
 
+# Disable the vendoring of libs when creating dist archives
+# disable-cargo-vendor = false
```

Comment:
Was not sure how to call that.
Will update the comment, should I change the configuration key name too?